src/common/actions/repositories.js

```diff
@@ -39,10 +41,9 @@ export function fetchUserRepositories(pageNumber) {
       if (result.status !== 'success') {
         throw getError(response, result);
       }
-      dispatch(setRepositories({
-        repos: result.payload.repos,
-        links: result.pageLinks
-      }));
+
+      // XXX deleted chunk should be handled in reducer?
+      dispatch(setRepositories(result));
```

Comment:
Handle the whole result in the reducer, rather than deconstructing it in the action creator. This keeps things simpler conceptually I think.